lib/api_auth/headers.rb

```diff
@@ -42,7 +42,7 @@ def timestamp
     def canonical_string
       [ @request.content_type,
         @request.content_md5,
-        @request.request_uri.gsub(/http:\/\/[^(,|\?|\/)]*/,''), # remove host
+        @request.request_uri.gsub(/https?:\/\/[^(,|\?|\/)]*/,''), # remove host
```

Comment:
This has also been fixed: https://github.com/mgomes/api_auth/blob/master/lib/api_auth/headers.rb#L56 (https://github.com/mgomes/api_auth/commit/839a7f2d01f9a6898eafabcf469dc84e8af67c90)